roles/ffmpeg/defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
-ffmpeg_path: /opt/install/ffmpeg_sources
\ No newline at end of file
+ffmpeg_path: "{{ project_base }}/install/ffmpeg_sources"
+x264_commit: a0cd7d38acb6c31973228ab207e18344920e0aa3
```

Comment:
This is the most recent commit on the stable branch.  I originally added it when trying to figure out why x264 wasn't building (turned out to be the linking issue), but it seems like a good idea anyway.